### ISSUE TITLE
Request for kinect camera parameters from real dataset

### ISSUE THREAD

**Issue by USTCWzy** [Healthcare-Robotics/bodies-at-rest#9]
Hello Henry,

I am very interested in your work and hope to re-train the network in both synthetic and real datasets. However, I notice that in .py file   evaluate_real_savefig.py line 285 self.kcam.init_from_yaml(osp.expanduser('~/catkin_ws/src/multimodal_pose/calibrations/kinect.yaml')), you use a .yaml file to initialize the camera parameters, which is not uploaded to the repositories. So I wonder if you could lend a hand and share the file.

Best wishes
Frank Wu

**Comment by USTCWzy** [comment]
And in line 846 
file_dir = "/media/henry/multimodal_data_2/CVPR2020_study/"+PARTICIPANT+"/data_checked_"+PARTICIPANT+"-"+POSE_TYPE, 
The directory of the real dataset containing .npy files(such as color.npy， depth_r.npy) also disappears in the bodies_at_rest_real dataset. would you mind sharing the related data files or explaining the relationship between those numpy files and uploaded .p files. 

Best wishes
Frank Wu

**Comment by henryclever** [comment]
I'm having trouble with the authentication pushing the kinect yaml, so here it is. I'll update the repo later. 
[kinect.zip](https://github.com/Healthcare-Robotics/bodies-at-rest/files/10805096/kinect.zip)

**Comment by henryclever** [comment]
> And in line 846 file_dir = "/media/henry/multimodal_data_2/CVPR2020_study/"+PARTICIPANT+"/data_checked_"+PARTICIPANT+"-"+POSE_TYPE, The directory of the real dataset containing .npy files(such as color.npy， depth_r.npy) also disappears in the bodies_at_rest_real dataset. would you mind sharing the related data files or explaining the relationship between those numpy files and uploaded .p files.
> 
> Best wishes Frank Wu

I will look into this -

**Comment by USTCWzy** [comment]
> I'm having trouble with the authentication pushing the kinect yaml, so here it is. I'll update the repo later. [kinect.zip](https://github.com/Healthcare-Robotics/bodies-at-rest/files/10805096/kinect.zip)

Thanks very much!

**Comment by henryclever** [comment]
OK - I took more a look into things. I had previously planned on deleting the `evaluate_real_savefig.py` file from the public repo because it's not necessary to generate the numerical results in the paper. All the numerical results can be had using `evaluate_real.py`. 

`evaluate_real_savefig.py` has code for producing images in the figures in the paper. For these, I used an earlier version of the dataset, which was saved in a different format (e.g. color.npy). All of these images should available in the data I published on Harvard Dataverse, but this code will have to be edited to accommodate for the new format. E.g. you access the rgb image by `rgb = dat['RGB'][im_num]`, where `dat` is a dict containing the data for a particular participant. 

I don't feel comfortable releasing the earlier dataset version because it is possible there is identifiable information in it (e.g. faces) that was not yet scrubbed for participants that did not want their face shown. I unfortunately don't have time resources to update the savefig file to accommodate for the public dataset right now, but if you do it and create a merge request I would be happy to review it and have you contribute to this project. 

-Henry
